src/modelfree/configs/multi/score.py

```diff
@@ -149,7 +149,7 @@ def save_activations(exp_name, score, spec):
         score['episodes'] = None
         # Trajectory length varies a lot between environments and opponents; make sure we have
         # a consistent number of data points.
-        score['timesteps'] = 5000
+        score['timesteps'] = 20000
```

Comment:
It feels like we might want to have a policy around when parameters like this in score_agent are changed, since it seems like ideally we wouldn't want any particular usage of score_agent to lean too much on the values of the default values here. That is to say, I don't have any particular objection to this value being the default, I just feel like changing these for reasons other than adding a new parameter or fixing an obvious bug is a bit of an bad pattern

Reply:
I totally agree for default configs in `score_agent`, but this named config feels fairly specific. It's already making other assumptions, e.g. about recording the feed-forward layer. If in the future we have experiments that need different numbers of timesteps then it might make sense to introduce an equivalent to `high_accuracy` and `medium_accuracy`, but it doesn't feel warranted right now.

Reply:
Ah, yep, I just misread and confirmation-biased myself into thinking these were modifications being directly made to score_agent